core/src/main/java/org/opensearch/sql/ast/expression/QualifiedName.java

```diff
@@ -110,4 +123,6 @@ public List<UnresolvedExpression> getChild() {
   public <R, C> R accept(AbstractNodeVisitor<R, C> nodeVisitor, C context) {
     return nodeVisitor.visitQualifiedName(this, context);
   }
+
+  public Boolean isMetadataField() { return Boolean.TRUE; }
```

Comment:
I'm confused by this line. Doesn't this always return true? was it meant to return `isMetadataField`?